vendor/k8s.io/kubernetes/pkg/features/kube_features.go

```diff
@@ -287,7 +287,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	AppArmor:                                    {Default: true, PreRelease: utilfeature.Beta},
 	DynamicKubeletConfig:                        {Default: false, PreRelease: utilfeature.Alpha},
 	ExperimentalHostUserNamespaceDefaultingGate: {Default: false, PreRelease: utilfeature.Beta},
-	ExperimentalCriticalPodAnnotation:           {Default: false, PreRelease: utilfeature.Alpha},
+	ExperimentalCriticalPodAnnotation:           {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
I know that I am responding late but yesterday, I realized that a side-effect of enabling this feature is kubelet will start preempting pods(https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/preemption/preemption.go) when there is a resource crunch on the node. This is when kubelet starts or when static pods are admitted onto the node but there is very good chance that some 'high priority'(not critical ones) pods are evicted from the node as preemption logic there is not priority aware.(except for critical pods).